task_4.py

```diff
@@ -0,0 +1,34 @@
+import sys
+from utils import print_intro
+
+print_intro()
+
+
+def get_input():
+    while True:
+        input_data = input("Enter an integer m or 'exit' to quit: ")
+        if input_data.lower() == 'exit':
+            sys.exit()
+        try:
+            m = int(input_data)
+            return m
+        except ValueError:
+            print("Error: Please enter an integer or 'exit' to quit.")
```

Comment:
I've found how to stop you program, but It should be divided to two separate functions.
the first one for "exit" functionality and the second one for casting of integer.
if you need some help with it, ping me